src/insns/sh4add_32bit.adoc

```diff
@@ -30,10 +30,16 @@ Encoding::
 ....
 
 pass:attributes,quotes[{cheri_cap_mode_name}] Description::
-Increment the address field of `cs2` by `rs1` shifted left by 4 bit positions. Clear the tag if the resulting capability is unrepresentable or `cs2` is sealed.
+Increment the address field of `cs2` by `rs1` shifted left by 4 bit positions
+and write the result to `cd`. The tag bit of the output capability is 0 if
+`cs2` did not have its tag set to 1, the incremented address is outside `cs2`
+'s <<section_cap_representable_check>> or `cs2` is sealed.
+
+include::malformed_shadd_clear_tag.adoc[]
```

Comment:
Maybe using something like this that can also be reused by cadd?:
```suggestion
and write the result to `cd`.

include::cap_add_representable_check.adoc[]
```

With `cap_add_representable_check.adoc` being:
```
The tag bit of the output capability is 0 if
`cs2` did not have its tag set to 1, the incremented address is outside `cs2`
's <<section_cap_representable_check>> or `cs2` is sealed.

NOTE: This instruction sets `cd.tag=0` if `cs2` 's bounds are
<<section_cap_malformed,malformed>>, or if any of the reserved fields are set.

```

Reply:
The operand is different between these instructions (cadd is cs1 and shxxx is cs2), so the include didn't work very well because adoc did not expand code ticks ` in macros. I will leave it like this